packages/headless/src/features/debug/ranking-info-parser.ts

```diff
@@ -0,0 +1,163 @@
+export type RankingInfo = ReturnType<typeof parseRankingInfo>;
```

Comment:
This is taken from the JSUI and adapted from underscore/StringUtils

Reply:
I think we could have some tests for `parseRankingInfo `. 

Nothing crazy, but I think we should have some we could copy/import from JSUI.

Reply:
Oh yes adding UTs in a separate PR, this one was getting too big